### DISCUSSION TITLE
[LionTooltip] Delay the opening

### DISCUSSION THREAD

**Discussion by MatthieuLebigre** [ing-bank/lion#1672] [Q&A]
Hello,

I have a question about the LionTooltip: What would be the better way to delay the opening of the tooltip ?
From what I understand, right now it is displayed as soon as the mouse is over the invoker, and there is no easy way to configure a delay before the opening.

Overriding the `_showMouse` could work, but is there a better way ? Could/should it be more generic to any mixin ?

**Comment by tlouisse** [comment]
Good question, the current design is that everything can be configured in the OverlayController, and the OverlayMixin in turn allows to configure the overlayController via `_defineOverlayConfig()`. 
Here is a POC branch that explores this concept: https://github.com/ing-bank/lion/blob/poc/overlayEnhancementsTooltip/packages/overlays/src/configurations/withTooltipConfig.js

@daKmoR @jorenbroekema opinions/ideas?
